Dockerfile

```
@@ -39,6 +40,13 @@ RUN apk add --no-cache git \
 WORKDIR /app
 RUN CGO_ENABLED=0 GOOS=linux go build
 
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+# Install ASCIIToSVG https://github.com/asciitosvg/asciitosvg
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+FROM golang:${A2S_GOLANG_BUILDER_TAG} as a2s-builder
+ARG A2S_VERSION=ca82a5c
+RUN GOBIN=/app go install github.com/asciitosvg/asciitosvg/cmd/a2s@${A2S_VERSION}
```

```suggestion
RUN GOBIN=/app go install github.com/asciitosvg/asciitosvg/cmd/a2s@"${A2S_VERSION}"
```

As the build arg can be specified with unexpected values, let's apply shellcheck good practises